dockstore-webservice/src/main/java/io/dockstore/webservice/resources/HostedToolResource.java

```diff
@@ -134,6 +134,7 @@ protected Tag getVersion(Tool tool) {
         tag.setAutomated(false);
         tag.setWdlPath("/Dockstore.wdl");
         tag.setReferenceType(Version.ReferenceType.TAG);
+        tag.setLastBuilt(new Date());
```

Comment:
If this is being set at creation time, isn't this redundant with dbCreateDate?

Reply:
Pretty much.  Wasn't sure if dbCreateDate/dbUpdateDate is something we're allowed to surface.

Reply:
I'm ok with it